apps/examples/smartfs/buffer_optimization/Kconfig_ENTRY

```diff
@@ -0,0 +1,3 @@
+config ENTRY_SMARTFS_BUF_OPTIMIZATION
+	bool "SmartFs Buffer Optimization Example"
```

Comment:
Example -> example

Also, can we make a uniform standard about where capital letters must be used?

Reply:
I think we need to check entire examples for what you mentioned. Let's merge as it is and then check all. How do you think?

Reply:
Thank you.